web/src/popover_menus.js

```diff
@@ -918,4 +931,258 @@ export function initialize() {
+            const left = ($("body").innerWidth() - $user_card_tippy.innerWidth()) / 2;
+            const top = ($("body").outerHeight() - $user_card_tippy.outerHeight()) / 2;
+            $user_card_tippy.css({
+                transform: `translate(${left}px, ${top}px)`,
+            });
+        }
+    }
+    const user_card_options = {
+        arrow: false,
+        popperOptions: {
+            modifiers: [
+                {
+                    name: "flip",
+                    options: {
+                        fallbackPlacements: "left",
+                    },
+                },
+            ],
+        },
+        onShow(instance) {
+            const {
+                user,
+                is_sender_popover = false,
+                has_message_context = false,
+                template_class = "user_popover",
+            } = instance.context;
+            on_show_prep(instance);
+            $(instance.popper).addClass("user_info_card_tippy " + template_class + "_tippy");
+
+            const args = popover_menus_data.render_user_info_popover(
+                user,
+                is_sender_popover,
+                has_message_context,
+                template_class,
+            );
+
+            const $bot_owner_element = $(".bot_owner");
+            if (
+                args.bot_owner &&
+                $bot_owner_element.prop("clientWidth") < $bot_owner_element.prop("scrollWidth")
+            ) {
+                $bot_owner_element.addClass("tippy-zulip-tooltip");
+            }
+
+            const content = $(render_user_info_popover_content(args)).get(0);
+            instance.setContent(parse_html(content.innerHTML));
+            instance.setProps({
+                popperOptions: {
+                    strategy: "fixed",
+                    modifiers: [
+                        {
+                            name: "preventOverflow",
+                            options: {
+                                altAxis: true,
+                                padding: {top: 10, bottom: $("#compose").outerHeight()},
+                            },
+                        },
+                        {
+                            name: "eventListeners",
+                            options: {
+                                scroll: false,
+                                resize: false,
+                            },
+                        },
+                    ],
+                },
+            });
+        },
+        onMount(instance) {
+            popover_instances.user_card = instance;
+            placement_on_mobile(instance);
+            const {user} = instance.context;
+            user_card_on_mount(instance, user);
+        },
+        onHidden(instance) {
+            $(".simplebar-content-wrapper").off("scroll");
+            instance.destroy();
+            popover_instances.user_card = undefined;
+            $(".tippy-overlay").remove();
```

Comment:
`$(".tippy-overlay")` that's a very generic class name to remove with `onHidden`. We definitely should localize the name to represent user card popover overlay.